src/python/bcc/libbcc.py

```diff
@@ -17,14 +17,15 @@
 lib = ct.CDLL("libbcc.so.0", use_errno=True)
 
 # keep in sync with bpf_common.h
+_MAP_CB_TYPE = ct.CFUNCTYPE(ct.c_int, ct.py_object, ct.c_void_p)
 lib.bpf_module_create_b.restype = ct.c_void_p
 lib.bpf_module_create_b.argtypes = [ct.c_char_p, ct.c_char_p, ct.c_uint]
 lib.bpf_module_create_c.restype = ct.c_void_p
 lib.bpf_module_create_c.argtypes = [ct.c_char_p, ct.c_uint,
-        ct.POINTER(ct.c_char_p), ct.c_int]
+        ct.POINTER(ct.c_char_p), ct.c_int, _MAP_CB_TYPE, ct.py_object]
 lib.bpf_module_create_c_from_string.restype = ct.c_void_p
 lib.bpf_module_create_c_from_string.argtypes = [ct.c_char_p, ct.c_uint,
-        ct.POINTER(ct.c_char_p), ct.c_int]
+        ct.POINTER(ct.c_char_p), ct.c_int, _MAP_CB_TYPE, ct.py_object]
```

Comment:
Does ctypes need the ct.py_object argument for this?

Reply:
This has been removed in the latest iteration of this PR.